recipes/pcre2/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "pcre2" %}
+{% set version = "10.30" %}
+{% set sha256 = "b549873a39f804480c2e6145a78adcba53e38162d90ef6ea92384f6ecf2fde76" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+
+test:
+  command:
+    - pcre2test -version
```

Comment:
Are there any other CLIs that we should be testing? Any libraries or headers that we should be checking for?